lib/redux/middleware/sentry_sdk_middleware.dart

```diff
@@ -11,6 +11,12 @@ class SentrySdkMiddleware extends MiddlewareClass<AppState> {
     if (action is ApiFailureAction) {
       Sentry.captureException(action.error, stackTrace: action.stackTrace);
     }
+    if (action is FetchAuthenticatedUserSuccessAction) {
+      Sentry.configureScope((scope) => scope.user = User(email: action.me.email, id: action.me.id, ipAddress: "{{auto}}"));
```

Comment:
Why do we need the `ipAddress: "{{auto}}"`? Maybe we can document this.

Reply:
Added a comment